src/components/About/About.js

```diff
@@ -1,24 +1,18 @@
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 
+import Blurbs from './Blurbs';
 
 export default class About extends React.Component {
   render() {
     return (
       <View>
-        <Text style={styles.header}>
-          our story
-        </Text>
+        <Blurbs />
```

Comment:
About/About.js: Code is clean and well-organized for future additions/amendments.